cmd/dkg.go

```diff
@@ -44,6 +44,7 @@ this command at the same time.`,
 	bindLogFlags(cmd.Flags(), &config.Log)
 	bindPublishFlags(cmd.Flags(), &config)
 	bindShutdownDelayFlag(cmd.Flags(), &config.ShutdownDelay)
+	bindPartialDepositsFlag(cmd.Flags(), &config.DepositAmounts)
```

Comment:
why are we introducing this flag in dkg command? In dkg we have `cluster-definition.json` file at the start and according to the design doc deposit amounts will be there in cluster definition. We can read the deposit amounts field from cluster definition and verify the values from there.

Reply:
This was also suggested by the design doc, but probably you are right. Need @xenowits to confirm if this was intentional or not.

Reply:
Resolved. Removed option from DKG. Amounts are specified in integer ETHs.